docs/styles/styles.css

```diff
@@ -189,6 +189,10 @@
     margin-block-start: var(--rh-space-2xl, 32px);
   }
 
+  rh-alert + div.grid {
```

Comment:
Does this need qualified with `div`?  could this just be `rh-alert + .grid`

Reply:
Resolved in [cea1ad3](https://github.com/RedHat-UX/red-hat-design-system/pull/1535/commits/cea1ad38d8a87443ebc633881699213a2ffc2f08).